Editors/RoomEditor.cpp

```diff
@@ -70,19 +65,35 @@ RoomEditor::RoomEditor(ProtoModelPtr model, QWidget* parent) : BaseEditor(model,
   connect(ui->currentViewComboBox, static_cast<void (QComboBox::*)(int)>(&QComboBox::currentIndexChanged),
           [=](int index) { viewMapper->setCurrentIndex(index); });
 
+  // filter the room preview scroll area to paint
+  // the checkerboard transparency pattern and to
+  // track the cursor position for the status bar
+  ui->roomPreview->widget()->installEventFilter(this);
+  // track the cursor position on the transparency pattern for the status bar
+  ui->roomPreview->widget()->setMouseTracking(true);
+  // also need to track it on the room preview itself so the event is received
+  ui->roomView->setMouseTracking(true);
+
+  cursorPositionLabel = new QLabel();
+  this->updateCursorPositionLabel(ui->roomPreview->widget()->cursor().pos());
+  assetNameLabel = new QLabel("obj_xxx");
```

Comment:
Is this a placeholder?

Reply:
yes that was there before. robert wrote that